.github/workflows/no-response.yml

```diff
@@ -0,0 +1,26 @@
+name: No Response
+
+## Closes issues marked with 'more information required' label after 7 days
+##    if original poster does not respond within that time
+
+on:
+  issue_comment:
+    types: [created]
```

Comment:
sorry dumb q: do you know what this means?

Reply:
Yeah, so on create of an issue comment (not edited/deleted). So if someone has a new comment